server/controllers/comment.js

```diff
@@ -81,6 +81,38 @@ class Comment {
     const updatedComment = await commentRepo.updateComment(newBody, id);
     return goodHttpResponse(response, 200, 'Comment updated', updatedComment);
   }
+
+  /** Create a new comment
+   * @param {object} request Request Object
+   * @param {object} response Response Object
+   * @returns {object} Comment Object
+   */
+  static async getCommentWithHistory(request, response) {
+    if (!request.isAuthorized) {
```

Comment:
is this endpoint only accessible to the whoever left the origin comment?

Reply:
Yes